docs/logs/discovery/tailing-log-files.md

```diff
@@ -0,0 +1,54 @@
+title: Shipping Log Files
+description: Shipping Log File Sources
```

Comment:
Maybe emphasize a bit the role of the type in the context of rules and how files of the unknown type are handled.